gff3tool/bin/gff3_to_fasta.py

```diff
@@ -1,4 +1,5 @@
-#! /usr/local/bin/python2.7
+#! /usr/local/bin/python3
+from __future__ import print_function
```

Comment:
It seems to me this line is not necessary.

Reply:
Yes, It's python3.  Python 3 already has the print function, so the from __future__ import print_function will have no necessary. Thank you.